### PR TITLE
[Insights Agent] Add test of the chart.

### PR DESCRIPTION
**Pull request by baderbuddy**
**Why This PR?**
Adds an automated test to the Insights Agent chart. The new values are not documented in the README because they only apply for CI/CD

Fixes #

**Changes**
Changes proposed in this pull request:

* Adds a new deployment and test pod for automated testing.
*

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Comment by sudermanjr** [comment]
Why not just use helm test hooks?

**Comment by baderbuddy** [comment]
Oh you had me doubting myself here :) I did on the Pod. The agent expects to be talking to Insights though along the way (from install instead of from when the tests start). So that's why there's a separate deployment for testing to mimic Insights.

**Review by rbren — APPROVED** [review]

File: stable/insights-agent/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 0.27.5
+version: 0.27.6
 appVersion: 1.12.1
```
Comment (makoscafee) [inline stable/insights-agent/Chart.yaml]: I think we are at `1.13.1` now

**Review by makoscafee — APPROVED** [review]
LGTM

**Pull request merged**